statsmodels/tsa/mstl.py

```diff
@@ -0,0 +1,240 @@
+"""
```

Comment:
Would be a good idea to reorg now with

tsa\
tsa\stl\
tsa\stl\__init__.py
tsa\stl\_stl.pyx
\tsa\stl\mstl.py